src/vs/platform/terminal/node/ptyHostService.ts

```diff
@@ -122,6 +124,7 @@ export class PtyHostService extends Disposable implements IPtyService {
 
 		// Setup logging
 		const logChannel = client.getChannel(TerminalIpcChannels.Log);
+		LogLevelChannelClient.setLevel(logChannel, getLogLevel(this._environmentService));
```

Comment:
Should this use `this._logService.getLevel()` instead of `getLogLevel(this._environmentService)`?

Reply:
yep